Projektdoku/Gespraeche.tex

```diff
@@ -0,0 +1,257 @@
+\item Vorschlag von Frau Hauptmann das Projekt als Web- oder Desktop-Anwendung umzusetzen.
+\item Team-intern wurde mittels einer Umfrage entschieden, das Projekt als Desktop-Anwendung zu realisieren.
+\end{itemize}
+
+\subsubsection{Termine}
+Den 16.5. als Termin für das 2. Meilensteintreffen vereinbart.
+
+
+
+\subsection{Zweites Meilensteintreffen 16.05.}
+
+Zeit: 14.15 - 15.15
+
+\subsubsection{Anwesenheit}
+\begin{itemize}
+	\item Frau Hauptmann
+	\item Leo Lindhorst
+	\item Oliver von Seydlitz
+	\item Denis Keiling
+	\item Hung Nguyen
+	\item Fabian Krehnke
+\end{itemize}
+
+\subsubsection{Inhalte}
+\begin{itemize}
+\item Vorstellung des Pflichtenheftes durch Oliver und der UI-Prototypen von Leo.
+\item Besprechung der nächsten Schritte, insbesondere Grob- u. Feinentwurf.
+\end{itemize}
+
+\subsubsection{Feedback}
+
+\paragraph{Pflichtenheft}
+\begin{itemize}
+\item Das Pflichtenheft wurde in Markdown erstellt und die enthaltenen
+Diagramme mittels Star-UML.
+\item Die Struktur des Pflichtenheftes muss überarbeitet werden damit es nachvollziehbar für den Kunden ist. Es gab bei der Vorstellung zu viele Sprünge innerhalb des Dokuments. Des Weiteren hatte das Dokument kein ausreichendes Inhaltsverzeichnis, welches ebenfalls überarbeitet werden soll.
+\end{itemize}
+
+\paragraph{Diagramme}
+\begin{itemize}
+\item Bei dem Systemkontext-Diagramm hat eine Legende zur Erklärung gefehlt.
+\item Bei den Anwendungsfall-Diagrammen soll das "'Include"' entfernt und zwischen Gruppe löschen und Gruppe anzeigen gesetzt werden.
+\end{itemize}
+
+\paragraph{UI-Protoypen}
+\begin{itemize}
+\item Die UI-Prototypen wurden mittels Affinity Designer gestaltet.
+\item Diskussion über die Darstellung in Listen oder Tabellen Form.
+\item Darstellung als Liste wird beibehalten.
+\item Besprechung der farblichen Kennzeichnung der Zustände und Festlegung der Zustände \textit{nicht gebucht} als Orange und \textit{gebucht} als Schwarz.
+\end{itemize}
+
+\subsubsection{Termine}
+
+25.05. Abgabe des Pflichtenheftes mit Änderungen
+
+
+\subsection{Drittes Meilensteintreffen 18.06.}
+
+Zeit: 17.30 - 19.00
+
+\subsubsection{Anwesenheit}
+\begin{itemize}
+	\item Frau Hauptmann
+	\item Leo Lindhorst
+	\item Oliver von Seydlitz
+	\item Denis Keiling
+	\item Hung Nguyen
+	\item Fabian Krehnke
+\end{itemize}
+
+\subsubsection{Inhalte} 
+
+\paragraph{Projektablaufplan}
+Vorstellung des Projektablaufplan von Leo.
+
+
+\paragraph{Datenbankentwurf}
+Vorstellung des Datenbankentwurfs von Hung. 
+
+
+\paragraph{Grob- und Fein- Entwurf}
+Software Entwurf Vorstellung/Erklärung von Denis.
+
+
+\subsubsection{Feedback}
+
+\paragraph{Projektablaufplan}
+\begin{itemize}
+\item Dieser wurde mittels Ganttproject erstellt. 
+\item Die Planung des Projektes war kompliziert da einige Prüfungstermine in Konflikt standen.
+\item Die Planung vergangener Termine ist nicht sinnvoll und führt zu inkonsistent.
```

Comment:
So war das Feedback nicht. Sie sagte, dass es nicht sinnvoll sei, den Plan Rückwirkend anders zu planen als es durchgeführt wurde. Zu Dokumentationszwecken und zum Zwecke der Vollständigkeit hatte sie aber gegen die Abbildung des Ist-Zustandes nichts einzuwenden oder?

Reply:
Nee, sie sagte nur, dass es nicht nötig sei. Mit Inkonsistenz hat das nichts zu tun

Reply:
okay dann hatte ich den Teil falsch verstanden.

nein das war in Ordnung